packages/hash/integration/jest.config.cjs

```diff
@@ -7,7 +7,6 @@ module.exports = {
     "!**/dist/**",
   ],
   coverageReporters: ["lcov", "text"],
-  preset: "ts-jest",
```

Comment:
I've replaced `ts-jest` with `babel` in this PR so that we can run tests, even if `tsc` doesn't compile for the entire backend. You can find more information [here](https://jestjs.io/docs/getting-started#using-babel).

Note that `tsc` compilation will still be checked in CI in the existing linting task.

cc @hashintel/backend @kachkaev

Reply:
Does `@babel/preset-typescript` autoload? It's a bit strange that the config file does not mention it, but if things work, all good!